fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/FDBSortQueryIndexSelectionTest.java

```diff
@@ -169,7 +170,6 @@ private void sortOnlyUnique(RecordMetaDataHook hook) throws Exception {
     /**
      * Verify that if the sort matches an index that can satisfy a filter that the index is used.
      */
-    @DualPlannerTest
```

Comment:
The new planner doesn't support `PlannableIndexTypes`. This didn't matter before, because we were ignoring the index type, but now that `fromIndexDefinition()` considers that it doesn't work.